vibi-dpu/src/pubsub/listener.rs

```diff
@@ -182,3 +172,44 @@ pub fn deserialized_data(message_data: &Vec<u8>) -> Option<Value> {
     );
     Some(deserialized_data)
 }
+
+async fn process_webhook_callback(deserialised_msg_data: &Value) -> bool {
```

Comment:
This is a misleading name. All this function does is update the pr_info in the sled-db if anything has changed, and return whether anything had changed. 
This is part of the pre-processing required to actually process the webhook-callback. 

```suggestion
async fn process_and_update_pr_if_different(deserialised_msg_data: &Value) -> bool {
```